lib/liquid/extensions.rb

```diff
@@ -7,6 +7,12 @@ def to_liquid
   end
 end
 
+class Symbol # :nodoc:
+  def to_liquid
+    self
```

Comment:
I'm fairly sure that everything that expects a string does `to_s` already. This is definitely the case in `StandardFilters`. (also `Symbol#size == Symbol#to_s.size`)

If you think it would be better to `to_s` here though, I considered that also and am not against it.